### PR TITLE
Correcting the connector length for tree diagram

### PR DESCRIPTION
**Pull request by ankityadav4**
## Current Behavior
TreeDiagramBefore1
TreeDiagramBefore2

## New Behavior

TreeDiagramAfter1

TreeDiagramAfter2

## Related Issue(s)

Fixes #

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/8d8c2765c6286c236983c2193c4d57f754ec2454'>8d8c2765c6286c236983c2193c4d57f754ec2454</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/react-charting/src/components/TreeChart/TreeChart.base.tsx [hunk 1]
```diff
@@ -169,9 +169,12 @@ class StandardTree {
...
   ): string {
     // gap adds ratio for parent.y to child.y
-    const path = `M${childX + rectWidth / 2},${childY - gap} H${parentX + rectWidth / 2} V${
-      parentY + rectHeight + gap / 2
-    }`;
+    const path =
+      parentX > childX
+        ? `M${childX},${childY - gap} H${parentX + rectWidth / 2}
+    V${parentY + rectHeight + gap / 2}`
+        : `M${childX + rectWidth},${childY - gap} H${parentX + rectWidth / 2}
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/TreeChart/TreeChart.base.tsx:176]: Avoid code duplication. Change the value used as rectWidth and use a single path definition

File: packages/react-charting/src/components/TreeChart/TreeChart.base.tsx [hunk 2]
```diff
@@ -167,14 +168,19 @@ class StandardTree {
...
     gap: number,
+    childXMax: number,
   ): string {
     // gap adds ratio for parent.y to child.y
-    const path =
-      parentX > childX
-        ? `M${childX},${childY - gap} H${parentX + rectWidth / 2}
-    V${parentY + rectHeight + gap / 2}`
-        : `M${childX + rectWidth},${childY - gap} H${parentX + rectWidth / 2}
+    // const path =
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/TreeChart/TreeChart.base.tsx:174]: remove commented code

Comment (ankityadav4) [reply packages/react-charting/src/components/TreeChart/TreeChart.base.tsx]: Removed the commented code

File: packages/react-charting/src/components/TreeChart/TreeChart.base.tsx (hunk 1 above)
Comment (ankityadav4) [reply packages/react-charting/src/components/TreeChart/TreeChart.base.tsx]: This is completed.

**Review by AtishayMsft — APPROVED** [review]

**Pull request merged**